saml2idp/base.py

```diff
@@ -222,10 +234,12 @@ def _validate_request(self):
                 doesn't match the one specified in the processor config.
         """
         request_acs_url = self._request_params['ACS_URL']
+
         if self._config['acs_url'] != request_acs_url:
-            raise exceptions.CannotHandleAssertion(
-                "couldn't find ACS url '{}' in SAML2IDP_REMOTES "
-                "setting.".format(request_acs_url))
+            msg = ("couldn't find ACS url '{}' in SAML2IDP_REMOTES "
+                   "setting.".format(request_acs_url))
+            self._logger.info(msg)
```

Comment:
Should we make this a `warning`? This could be a big timesaver for someone :smiley